storage/simulation/cluster.go

```diff
@@ -134,13 +216,12 @@ func (c *Cluster) String() string {
 		}
 	}
 
-	var nodeIDs []int
+	var nodeIDs []proto.NodeID
 	for nodeID := range c.nodes {
-		nodeIDs = append(nodeIDs, int(nodeID))
+		nodeIDs = append(nodeIDs, nodeID)
 	}
-	sort.Ints(nodeIDs)
+	sort.Sort(proto.NodeIDSlice(nodeIDs))
```

Comment:
It works because `sort.Sort()` acts on the underlying array instead of the slice object. But declaring `var nodeIDs` NodeIDSlice` is probably a little nicer than casting at the call to `Sort`.